dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -61,6 +61,12 @@ def next
   def show
     authorize! :read, ScriptLevel
     @script = Script.get_from_cache(params[:script_id])
+
+    if @script.redirect_to?
+      redirect_to build_script_level_path(Script.find_by_name(@script.redirect_to).starting_level)
```

Comment:
Is the DB dip intentional (necessary) or should we be hitting the cache (if caching is enabled)?